lms/templates/base.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='my.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='prism.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='hawcons.css') }}">
+    <link rel="manifest" href="{{ url_for('static', filename='manifest.json') }}">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="transparent">
+    <link rel="apple-touch-icon" sizes="57x57" href="{{ url_for('static', filename = 'icons/apple-icon-57x57.png')}}">
```

Comment:
This is a very large payload. We should allow only 180x180. See [this](https://stackoverflow.com/a/21144916/1058671) StackOverflow answer.

Reply:
Ok will look into it!